app/models/clientsModel.js

```diff
@@ -41,7 +41,31 @@ async function getEmailByJoinJobs(jobId) {
     }
 }
 
+/**
+ * Toggles client's notifications given their id.
+ * 
+ * @param {*} clientId id of the client
+ * @returns {boolean} Whether the client's notifications are on or off.
+ */
+async function toggleClientNotification(clientId){
+    try {
+        const clients = await db('clients')
```

Comment:
You don't have to get the client first, then revert the boolean, this can be done with one query

`UPDATE table SET boolean_field = NOT boolean_field WHERE id = :id`

Find out what the correct syntax for the above query is in knex.